src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -286,68 +357,78 @@ function Wait-Function
     }
 }
 
-
 <#
 .SYNOPSIS
-Waits for specified duration if not-mocked, otherwise skips wait.
+Wait for specified duration if not-mocked, otherwise skips wait.
 
 .PARAMETER timeout
 Timeout in seconds
 #>
 function Wait-Seconds
 {
-    param([int] $timeout)
-    
-    [Microsoft.WindowsAzure.Testing.TestUtilities]::Wait($timeout * 1000)
-}
+    param(
+        [int] $timeout
+    )
 
+    Start-Sleep -Seconds $timeout
```

Comment:
@hovsepm  It seems that this issue has come up before (see: https://github.com/Azure/azure-powershell/pull/1846#discussion_r53832755). I have updated my pull request to use the workaround recommended by @markcowl here: (https://github.com/Azure/azure-powershell/pull/1846#issuecomment-188571285).